lib/install.ps1

```
@@ -533,23 +533,131 @@ function persist_permission($manifest, $global) {
+    $running_processes = $test_result.RunningProcesses
+    $servicesToStop = $test_result.ServicesToStop
+    $processdir = $test_result.ProcessDir
+
+    $stopped_services = @()
+    $stopped_processes = @()
+    $blocked = $test_result.Blocked
+
+    if ($running_processes.Count -gt 0) {
+        if ($forcekill) {
+            warn "The following instances of `"$app`" are still running. Scoop is configured to force kill them."
+            Write-Host ($running_processes | Out-String)
```

PSScriptAnalyzer (1.24.0)

[warning] File 'install.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)